versions.txt

```diff
@@ -2,24 +2,24 @@
 # by default with the OpenTelemetry Operator. This would usually be the latest
 # stable OpenTelemetry version. When you update this file, make sure to update the
 # the docs as well.
-opentelemetry-collector=0.122.1
+opentelemetry-collector=0.123.1
```

Comment:
Should it be 0.123.0?

Reply:
This should be the version on the released distributions, which is [0.123.1](https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/v0.123.1).